Formula/mesa.rb

```diff
@@ -9,8 +9,8 @@ class Mesa < Formula
   head "https://gitlab.freedesktop.org/mesa/mesa.git"
 
   livecheck do
-    url "https://archive.mesa3d.org/"
-    regex(/href=.*?mesa[._-]v?(\d+(?:\.\d+)+)\.t/i)
+    url "https://www.mesa3d.org/news/"
+    regex(/Mesa v?(\d+(?:\.\d+)+) is released/i)
```

Comment:
Since the start/end of the regex isn't restricted, this could technically match something unexpected like "when Mesa 1.2.3 is released" (referring to a future release). It's unlikely to happen since the news page is pretty standardized but I usually like to err on the side of caution.

If we target the `<h2>Mesa 1.2.3 is released</h2>` titles, we can use the angle brackets from the `h2` elements to anchor the start/end of the matched text (making sure to account for optional whitespace).

```suggestion
    regex(/>\s*Mesa v?(\d+(?:\.\d+)+) is released\s*</i)
```